frontend/packages/console-shared/src/promql/resource-metrics.ts

```diff
@@ -41,6 +42,9 @@ const podControllerMetricsQueries = {
   [ResourceUtilizationQuery.CPU]: _.template(
     "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{} * on(pod) group_left(workload, workload_type) namespace_workload_pod:kube_pod_owner:relabel{workload='<%= name %>', workload_type='<%= type %>'}) by (pod)",
   ),
+  [ResourceUtilizationQuery.CPU_ARCH]: _.template(
+    "sum by(node) (kube_node_info{node='<%= name %>'})",
+  ),
```

Comment:
Do we still need it?

Reply:
Yes this was not required now , so removed it.